hive/Dockerfile

```diff
@@ -122,15 +115,15 @@ RUN microdnf update && \
 USER stackable
 WORKDIR /stackable
 
-COPY --link --from=builder /stackable/apache-hive-${PRODUCT} /stackable/apache-hive-${PRODUCT}
-RUN ln -s /stackable/apache-hive-${PRODUCT}/ /stackable/hive
+COPY --chown=stackable:stackable --from=builder /stackable/apache-hive-metastore-${PRODUCT}-bin /stackable/apache-hive-metastore-${PRODUCT}-bin
```

Comment:
Does the build brake with `--link` ? If not I would suggest to keep it as it *should* be more performant

Reply:
I can only say the previous version caused failure and the current one seems to be fine.
We might be able to use `COPY --link --chown=stackable:stackable --from=builder /stackable/apache-hive-metastore-${PRODUCT}-bin /stackable/apache-hive-metastore-${PRODUCT}-bin`, but we would need to start the whole CI testing stuff again. Would it be ok to merge this to hopefully get the CI green and than try to improve the performance afterwards if desired?

Reply:
yeah! That's allright. I would appreciate a ticket or a comment in the dockerfile to not forget it :)

Reply:
Thanks :)